### PR TITLE
highlight user details link

### PR DESCRIPTION
**Pull request by denyskon**
This PR adds a separated column in the users table for operations. The username link now redirects back to user page.

grafik

Resolves https://github.com/go-gitea/gitea/pull/26713#pullrequestreview-1603001285

Co-authored-by: silverwind <me@silverwind.io>

### CODE REVIEW

**Comment by wxiaoguang** [comment]
TBH ... IMO ... such tooltip seems redundant .......

**Comment by silverwind** [comment]
We should at least make user and org lists behave the same. User list name click goes to user details, org list name click goes to org page.

**Comment by denyskon** [comment]
I'd do an org details page, but aren't we in feature freeze right now?

**Comment by silverwind** [comment]
We should be as per recent communication but until the `release/v1.21` branch is forked, we aren't in fact.

**Comment by silverwind** [comment]
Maybe a home link column for both tables would be good to somewhat solve the inconsistency. I think there is still space in the table for a small "Home" column at the end with https://primer.github.io/octicons/home-16 icon.

**Comment by delvh** [comment]
(My original suggestion was an `info` icon such as `octicon-info` by the way)

**Comment by silverwind** [comment]
Maybe https://primer.github.io/octicons/person-fill-16 for the user page link?

**Comment by denyskon** [comment]
grafik

I somehow tried to implement both suggestions. The username link's destination is the user page, and the user icon in the user details column leads to the user details page.

**Comment by silverwind** [comment]
So I did a few tweaks, maybe you like them:

Screenshot 2023-09-29 at 20 44 28
Screenshot 2023-09-29 at 20 45 05

The home icon looked odd there and I see you retained the user name linking to home, so I removed it.

If you prefer, the user name could like to details page and then a home button would be necessary, but I prefer it the way it currently is because it's consisten with org page. In both cases I renamed last column to "Actions" (with a separate translation to distinguish from Gitea Actions).

**Review by silverwind — APPROVED** [review]

**Comment by denyskon** [comment]
@silverwind Maybe operations would be a better word for it? I think actions might be confusing for translation. But otherwise, I think your idea is nice, thanks :)

**Comment by denyskon** [comment]
Operations is the standard term since https://github.com/go-gitea/gitea/pull/22067

**Comment by silverwind** [comment]
It's such a long word, but okay.

File: templates/admin/org/list.tmpl [hunk 1]
```diff
@@ -25,7 +25,7 @@
 							{{ctx.Locale.Tr "admin.users.created"}}
 							{{SortArrow "recentupdate" "leastupdate" $.SortType false}}
 						</th>
-						<th>{{ctx.Locale.Tr "admin.users.edit"}}</th>
+						<th>{{ctx.Locale.Tr "repo.commit.operations"}}</th>
```
Comment (delvh) [inline templates/admin/org/list.tmpl]: See https://github.com/go-gitea/gitea/pull/27354#discussion_r1341734575

File: templates/admin/user/list.tmpl [hunk 2]
```diff
@@ -77,14 +77,15 @@
 							{{ctx.Locale.Tr "admin.users.last_login"}}
 							{{SortArrow "lastlogin" "reverselastlogin" $.SortType false}}
 						</th>
+						<th>{{.locale.Tr "repo.commit.operations"}}</th>
```
Comment (delvh) [inline templates/admin/user/list.tmpl]: See above

File: templates/admin/org/list.tmpl (hunk 1 above)
Comment (silverwind) [reply templates/admin/org/list.tmpl]: Yeah let's remove these.

**Review by silverwind — APPROVED** [review]

**Review by delvh — APPROVED** [review]

File: templates/admin/user/list.tmpl [hunk 3]
```diff
@@ -105,6 +106,12 @@
 							{{else}}
 								<td><span>{{ctx.Locale.Tr "admin.users.never_login"}}</span></td>
 							{{end}}
+							<td>
+								<div class="gt-df gt-gap-3">
+									<a href="{{$.Link}}/{{.ID}}" data-tooltip-content="{{ctx.Locale.Tr "admin.users.details"}}">{{svg "octicon-person"}}</a>
```
Comment (lunny) [inline templates/admin/user/list.tmpl:111]: How about also use `{{.HomeLink}}`?

File: templates/admin/user/list.tmpl [hunk 4]
```diff
@@ -105,6 +106,12 @@
...
+									<a href="{{$.Link}}/{{.ID}}/edit" data-tooltip-content="{{ctx.Locale.Tr "edit"}}">{{svg "octicon-pencil"}}</a>
```
Comment (lunny) [inline templates/admin/user/list.tmpl:112]: As above

File: templates/admin/user/list.tmpl (hunk 3 above)
Comment (wxiaoguang) [reply templates/admin/user/list.tmpl:111]: It needs `admin` link, but not `home` link.

**Review by lunny — APPROVED** [review]

**Pull request merged**